cudamapper/src/main.cu

```diff
@@ -122,6 +125,7 @@ struct ApplicationParameteres
     std::int32_t min_bases_per_residue          = 100;  // b
     float min_overlap_fraction                  = 0.95; // z
     bool all_to_all                             = false;
+    bool perform_overlap_end_rescue             = false;
```

Comment:
So easier understanding add `// R` and move above `bool all_to_all`

Reply:
Fixed in 0c393c8be.